### DISCUSSION TITLE
Simple Installation Guide

### DISCUSSION THREAD

**Discussion by BerndCzech** [Akkudoktor-EOS/EOS#134] [Ideas]
When I opened the repo I saw it is about EnergyOptimization.

What I did not see is how I could make it work.

* Does it work on **my PV** system?
* What **hardware** components do I need (raspi, esp32) ???
* where is the **docker compose** file :smile: ?

Is there anybody else thinking a major priority should be to make all the work more accessible?  If so I would start working towards that and file pull requests.

In my opinion the knowledge in this repo would scale better across dummies like me.

**Comment by drbacke** [comment]
I think a docker Release File would be very helpfull. I'm working on a swagger FastAPI Server, so the API will be better documented. 
Unit Testing is not finished ....
It's way to early, to think about a productive Integration.
